code/stable/dblpendulum/src/cpp/Calculations.hpp

```diff
@@ -1,6 +1,7 @@
 /** \file Calculations.hpp
     \author Dong Chen
     \brief Provides functions for calculating the outputs
+    \Generated by Drasil Version: "v0.1-alpha"
```

Comment:
Since you have the colon to separate the variable component of the text, you can probably remove the quotation marks around the version information. Though I might prefer "Generated by Drasil v0.1-alpha". Maybe format of the information message can be variable too?

Reply:
Ok, I will change that right now!